config/config.go

```diff
@@ -461,6 +465,15 @@ func Init() error {
 		Parameters.SyncMode = SyncMode
 	}
 
+	if Parameters.SyncStateMaxThread == 0 {
+		syncStateMaxPeer := uint32(float64(memory.TotalMemory() / defaultSyncStateThreadMemory))
```

Comment:
L469. 470 and `defaultSyncStateMaxPeer` still have the old name

Reply:
Fixed